mos-platform/dodo/api.h

```diff
@@ -0,0 +1,144 @@
+#pragma once
```

Comment:
The `#pragma once` here seems redundant with the include guards. Was there a particular reason to use both?

Reply:
I got this from another project I've worked on where I guess the idea is that the include guard is more portable but pragma once is faster on some compilers (allegedly), so each has a benefit. Looking through some headers in other platforms I see a possible preference for include guards, so I'll go with that. I don't have a strong opinion.